wiki/Tournaments/CES/CES2/en.md

```diff
@@ -0,0 +1,119 @@
+---
+tags:
+  - CES
+---
+
+# CES League osu! Tournament 2
+
+![CES League osu! Tournament 2 Logo](img/logo.jpg)
```

Comment:
```suggestion
![CES League osu! Tournament 2 logo](img/logo.jpg)
```
Dude you forgot the image extension is png. (or you can convert the png to jpg which is better if the image doesn't have any transparency element)